src/SymbolEditor/SymbolEditorApp/MainWindow.xaml

```diff
@@ -29,124 +38,184 @@
+                                        </Grid>
+                                    </StackPanel>
+                                    <ControlTemplate.Triggers>
+                                        <Trigger Property="TreeViewItem.IsExpanded" Value="False">
+                                            <Setter TargetName="ItemsHost" Property="UIElement.Visibility" Value="Collapsed" />
+                                        </Trigger>
+                                        <Trigger Property="ItemsControl.HasItems" Value="False">
+                                            <Setter TargetName="Expander" Property="UIElement.Visibility" Value="Hidden" />
+                                        </Trigger>
+                                        <Trigger Property="TreeViewItem.IsSelected" Value="True">
+                                            <Setter TargetName="Bd" Property="Panel.Background" Value="{DynamicResource MahApps.Brushes.Accent}" />
+                                            <Setter TargetName="Bd" Property="Border.BorderBrush" Value="{DynamicResource {x:Static SystemColors.ActiveBorderBrushKey}}" />
+                                            <Setter Property="TextElement.Foreground" Value="{DynamicResource {x:Static SystemColors.HighlightTextBrushKey}}" />
+                                        </Trigger>
+                                        <Trigger Property="TreeViewItem.IsMouseOver" Value="True">
+                                            <Setter TargetName="Bd" Property="Border.BorderBrush" Value="{DynamicResource MahApps.Brushes.Accent}" />
+                                        </Trigger>
+                                        <MultiTrigger>
+                                            <MultiTrigger.Conditions>
+                                                <Condition Property="TreeViewItem.IsSelected" Value="True" />
+                                                <Condition Property="Selector.IsSelectionActive" Value="False" />
+                                            </MultiTrigger.Conditions>
+                                            <Setter TargetName="Bd" Property="Panel.Background" Value="{DynamicResource MahApps.Brushes.WindowTitle.NonActive}" />
+                                        </MultiTrigger>
+                                        <Trigger Property="UIElement.IsEnabled" Value="False">
+                                            <Setter Property="TextElement.Foreground" Value="{DynamicResource {x:Static SystemColors.GrayTextBrushKey}}" />
+                                        </Trigger>
+                                    </ControlTemplate.Triggers>
+                                </ControlTemplate>
+                            </Setter.Value>
+                        </Setter>
+                    </Style>
+                </esri:TableOfContents.ItemContainerStyle>
+            </esri:TableOfContents>
+            <TextBlock
```

Comment:
I added a label here because I wasn't expecting the symbol editing functionality to require right-clicking the TOC. This is the only substantive change to this file besides namespace updates (TOC moved out of the Toolkit preview package)